pkg/build/admission/strategyrestrictions/admission.go

```diff
@@ -156,7 +156,7 @@ func (a *buildByStrategy) checkBuildRequestAuthorization(req *buildapi.BuildRequ
 func (a *buildByStrategy) checkAccess(strategy buildapi.BuildStrategy, subjectAccessReview *authorizationapi.LocalSubjectAccessReview, attr admission.Attributes) error {
 	resp, err := a.client.LocalSubjectAccessReviews(attr.GetNamespace()).Create(subjectAccessReview)
 	if err != nil {
-		return err
+		return admission.NewForbidden(attr, err)
```

Comment:
This will already be a status error, but the user was requesting a different API endpoint, so the error would look weird.